frontend/src/services/waste.service.js

```diff
@@ -0,0 +1,53 @@
+import HelperService from "@/services/helper.service";
+
+class WasteService {
+  /**
+   * Returns all waste entry that match the filters (args). If args is empty, all waste entry will be returned.
+   * @param args Dictionary with the filters.
```

Comment:
Please add the allowed filters to the comments.

Reply:
added filters to comments